### ISSUE TITLE
Accessibility - OpenY rose, frontpage

### ISSUE THREAD

**Issue by podarok** [ymcatwincities/openy#1085]
One line - one task.
http://sandbox.openymca.org/

`ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|div||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.08:1. Recommendation: change text colour to #fff.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail|h2|#block-useraccountmenu-menu|This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.08:1. Recommendation: change background to #00192e.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G145.Fail|h2|#block-mainnavigation-menu|This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 3:1, but text in this element has a contrast ratio of 1.08:1. Recommendation: change background to #00192e.
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_1.F77|div|#block-openy-rose-branding|Duplicate id attribute value "block-openy-rose-branding" found on the web page.
ERROR|WCAG2AA.Principle4.Guideline4_1.4_1_2.H91.A.EmptyNoId|a||Anchor element found with no link content and no name and/or ID attribute.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.86:1. Recommendation: change background to #a26b00.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.53:1. Recommendation: change background to #007faf.
ERROR|WCAG2AA.Principle1.Guideline1_1.1_1_1.H37|img||Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
ERROR|WCAG2AA.Principle1.Guideline1_1.1_1_1.H37|img||Img element missing an alt attribute. Use the alt attribute to specify a short text alternative.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 1.37:1. Recommendation: change background to #597b8d.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.13:1. Recommendation: change background to #008575.
ERROR|WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail|span||This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.39:1. Recommendation: change background to #d84009.`

**Comment by ymcagtc** [comment]
Assigning to @podarok as this is under development
